io/disk/smartctl.py.data/Readme

```diff
@@ -0,0 +1,11 @@
+smartctl - controls  the Self-Monitoring, Analysis and Reporting Technology (SMART) system built into most ATA/SATA and SCSI/SAS hard drives and solid-state drives 
+
+Values to be passed in yaml file:
+
+* Name of the devices on which this scripts should be run.
+
+* smartctl tool options.
+
+Note: 
+1. Please do not input OS drives. 
```

Comment:
Can you add a check in the test for this, and skip in such cases ?
`df -h /boot` helps here